infrastructure/quay_alb.tf

```diff
@@ -98,4 +95,42 @@ resource "aws_alb_target_group" "quay_alb_metrics_target_group" {
   }
 }
 
-/* TODO: Add IPs of ELB automatically to the target group */
+# Use this to get the resource of the ELB using the tag created by k8s 
+data "aws_resourcegroupstaggingapi_resources" "quay_elb" {
```

Comment:
the service is not created until we apply the generated yaml file. So this is a circular dependency

Reply:
One solution to this would be to include the `kubectl apply` in the terraform logic. Having a `null_resource` dependency with local-exec should allow you to do that 

https://developer.hashicorp.com/terraform/language/resources/provisioners/local-exec